pvanalytics/quality/outliers.py

```diff
@@ -95,6 +96,7 @@ def hampel(data, window=5, max_deviation=3.0, scale=1.4826):
     median = data.rolling(window=window, center=True).median()
     deviation = abs(data - median)
     mad = data.rolling(window=window, center=True).apply(
-        stats.median_absolute_deviation
+        robust.scale.mad,
+        kwargs={'c': 1.0/scale}
```

Comment:
Commenting here: let's go with scale=None as the default, and not pass 'c' if scale=None.

Reply:
For the docstring, suggest
```
Scale factor used to estimate the standard deviation as
:math:`MAD / scale`. If scale=None (default), then the scale
factor is taken to be `scipy.stats.norm.ppf(3/4.)` (approx. 0.6745),
and :math:`MAD / scale` approximates the standard deviation
of Gaussian distributed data.

Reply:
Sounds good.